tlspuffin/src/integration_tests/mod.rs

```diff
@@ -42,6 +42,6 @@ fn version_test() {
     assert!(version.expect("missing version string").contains("5.3.0"));
     #[cfg(feature = "wolfssl540")]
     assert!(version.expect("missing version string").contains("5.4.0"));
-    #[cfg(feature = "wolfssl550")]
```

Comment:
Why removing wolfssl550?

Reply:
We actually stop at 5.4.0, so this was triggering a warning flagging this as a non-existing feature. I think I'll write a more generic version of this test once we merge the dynamic PUT registration process.